ios/JSProperties.swift

```diff
@@ -13,6 +13,7 @@ enum JSProperties: String {
     case programType
     case options
     case consentText
+    case cardConfig
```

Comment:
To be consistent with Android, I would suggest renaming `JSProperties` with `FidelSetupProperties`. Maybe on Android as well: `FidelSetupKeys` => `FidelSetupProperties`. And then move the CardConfig child enum into a separate enum.